### ISSUE TITLE
Failure when candidate already installed

### ISSUE THREAD

**Issue by yogeek** [gantsign/ansible_role_sdkman_init#20]
Hi and thanks for this role,
When I launch ansible twice, I get an error if sdkman_init role has already passed successfully the first time :

```
TASK [gantsign.sdkman_init : install candidates] *******************************
failed: [default] (item=[{u'username': u'dev'}, {u'version': u'', u'candidate': u'java'}]) => {"ansible_loop_var": "item", "changed": false, "cmd": "source ~/.sdkman/bin/sdkman-init.sh && sdk install java  \n", "delta": "0:00:01.270544", "end": "2020-04-05 23:39:12.001655", "failed_when_result": true, "item": [{"username": "dev"}, {"candidate": "java", "version": ""}], "msg": "non-zero return code", "rc": 1, "start": "2020-04-05 23:39:10.731111", "stderr": "", "stderr_lines": [], "stdout": "\n\u001b[1;31mStop! java 11.0.6.hs-adpt is already installed.\u001b[0m", "stdout_lines": ["", "\u001b[1;31mStop! java 11.0.6.hs-adpt is already installed.\u001b[0m"]}
```

By looking at the code, it seems to come from [this line](https://github.com/gantsign/ansible_role_sdkman_init/blob/master/tasks/main.yml#L12).
Indeed, when I try to launch the `sdk install java` command on a VM if the latest candidate is already installed, the error code returned is 1... so the `failed_when` test is true because of the first part of the condition.

It seems to be caused by [this change](https://github.com/sdkman/sdkman-cli/commit/1fe19fefaef47a65aa01374abb9c39bbe0cabbcb#diff-9a343c587776521dfc67157d1cacfa6f) on sdkman code.

**Comment by yogeek** [comment]
@freemanjp Thanks for the fix. 
Is the role automatically pushed to Galaxy ? Because when I do `ansible-galaxy role install -r requirements.yml` with the following requirements, I still have the old version.

```
cat requirements.yml
- name: gantsign.sdkman
- name: gantsign.sdkman_init

``` 

In Galaxy site, I see that the role has been updated 2 hours ago but the version still look the same...
How does the relase to galaxy process work please ?

**Comment by freemanjp** [comment]
@yogeek thanks for reporting this issue. Sorry, the release took so long, I've been having a little trouble with the Travis CI config. You can find the fix in the [1.2.0](https://github.com/gantsign/ansible_role_sdkman_init/releases/tag/1.2.0) release of this role.
